homeassistant/components/nut/const.py

```diff
@@ -216,6 +217,18 @@
         "mdi:flash",
         None,
     ],
+    "ambient.humidity": [
+        "Ambient Humidity",
+        PERCENTAGE,
+        "mdi:water-percent",
```

Comment:
```suggestion
        None,
```

Reply:
Icon and device class are mutually exclusive properties.  We should not set an icon if the device class is set.

Reply:
mhhh ... should I than also correct properties for `ups.temperature` , `ups.realpower` , `ups.realpower.nominal` , `battery.charge` and `battery.temperature` ? :smiley: